doc/v2/documentation_data_format.rst

```diff
@@ -575,41 +577,61 @@ Detailed field description
 
 - ``initializationPriorType`` [STRING, OPTIONAL]
 
-  Prior types used for sampling of initial points for estimation. Sampled
-  points are clipped to lie inside the parameter boundaries specified by
-  ``lowerBound`` and ``upperBound``. Defaults to ``parameterScaleUniform``.
+  Prior types used for sampling initial points for estimation. If the prior is
+  not truncated (see ``initializationPriorParameters``), sampled points are
+  clipped to lie within the parameter boundaries specified by ``lowerBound``
+  and ``upperBound``. Otherwise, points are sampled directly from the truncated
+  distribution. The default is ``parameterScaleUniform``.
 
-  Possible prior types are:
+  Possible prior types are (for mathematical definition see below):
 
-    - *uniform*: flat prior on linear parameters
-    - *normal*: Gaussian prior on linear parameters
+    - *cauchy*, Cauchy prior on linear parameters
+    - *chisquare*, Chisquare prior on linear parameters
+    - *exponential*, Exponential prior on linear parameters
+    - *gamma*, Gamma prior on linear parameters
     - *laplace*: Laplace prior on linear parameters
     - *logNormal*: exponentiated Gaussian prior on linear parameters
-    - *logLaplace*: exponentiated Laplace prior on linear parameters
+    - *normal*: Gaussian prior on linear parameters
+    - *rayleigh*, Rayleigh prior on linear parameters
+    - *uniform*: Flat prior on linear parameters
+    - *parameterScaleCauchy*, Cauchy prior on original parameter scale
```

Comment:
I would add the `parameter*` to a new list and start it with something like:
Supported `parameterScales` are: 
- *parameterScaleLaplace*: Laplace prior on original parameter scale
...